### ISSUE TITLE
Invalid characters in localisation files

### ISSUE THREAD

**Issue by CaligulaCaesar** [tboby/cwtools-stellaris-config#212]
This is a common newbie modder error (or simply careless copying between Word and localisation files). The following characters are confirmed to be invalid characters:

„“ ‚‘ – ” ’

They show as "?" ingame, which it is obviously best to avoid! Let me know when you want to add this feature, I will test a few more characters.

**Comment by Sciencmine** [comment]
Nope, they are punctuation marks for other languages and are displayed well in those languages.

**Comment by 17blue17** [comment]
I am closing all old issues as I am not sure what is current and what is not.  Please create a new report if needed.
